deployment/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
```

Comment:
Hmm. We'll have to test that thoroughly on stage. Is this required?

Reply:
It's definitely the path of least resistance to get the latest version of Inkscape (0.92.3 which is also the version I used in my testing).

Reply:
Using a ppa shouldn't be too bad:

http://ubuntuhandbook.org/index.php/2017/01/install-inkscape-0-92-ppa-ubuntu-16-04-16-10-14-04/

----

I'm not against bumping Ubuntu, but we can't just casually bump it here w/o some thorough manual testing.

Reply:
I understand your concern. Let me try to compile Inkscape or install from a deb package.

Reply:
> Using a ppa shouldn't be too bad:

That did the trick thank you :+1:  

Commit b6daae1 reverted our container to `ubuntu:xenial` and install Inkscape from a PPA